python/lsst/afw/image/exposure/_exposureContinued.py

```diff
@@ -31,6 +31,10 @@
 from ..image._fitsIoWithOptions import imageReadFitsWithOptions, exposureWriteFitsWithOptions
 from ._exposure import ExposureI, ExposureF, ExposureD, ExposureU, ExposureL
 from .exposureUtils import bbox_to_convex_polygon, bbox_contains_sky_coords
+from lsst.pex.exceptions import InvalidParameterError
+import lsst.geom as geom
+
+sigma2fwhm = 2.*np.sqrt(2.*np.log(2.))
```

Comment:
Module variables used as module global constants are UPPERCASE_WITH_UNDERSCORES.  https://developer.lsst.io/python/style.html#naming-conventions-for-science-pipelines

Reply:
Addressed in #f56a622f1